drools-eclipse/org.drools.eclipse/src/main/resources/org/drools/eclipse/wizard/project/sample.bpmn.template

```diff
@@ -11,7 +11,7 @@
              xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
              xmlns:tns="http://www.jboss.org/drools">
 
-  <process id="com.sample.bpmn.hello" tns:packageName="defaultPackage" name="Hello World" isExecutable="true" processType="Private">
+  <process id="com.sample.bpmn.hello" tns:packageName="defaultPackage" name="Hello World" isExecutable="true" processType="Private" tns:packageName="defaultPackage">
```

Comment:
@jomarko apologies for the late response, but isn't this adding the sane property that is already available on the same line (second place, after id)?